src/shared/components/Pagination/Pagination.tsx

```diff
@@ -7,25 +8,51 @@ import { ChevronButton, PaginationButton, PaginationWrapper, ThreeDotsWrapper }
 export type PaginationProps = {
   itemsPerPage?: number
   totalCount?: number
-  maxPaginationLinks?: number
+  defaultPaginationLength?: number
   onChangePage: (page: number) => void
   page: number
 }
 
+const PAGINATION_BUTTON_WIDTH = 48
+const VIEWPORT_PADDING = 32
```

Comment:
If those apply to button styles, we should also use these values in `Pagination.styles`, wdyt?